sysdump/sysdump_test.go

```diff
@@ -145,6 +145,21 @@ type fakeClient struct {
 	nodeList *corev1.NodeList
 }
 
+func (c *fakeClient) ListCiliumClusterwideEnvoyConfigs(ctx context.Context, opts metav1.ListOptions) (*ciliumv2alpha1.CiliumClusterwideEnvoyConfigList, error) {
+	//TODO implement me
```

Comment:
Nit: remove these TODO comments, the `panic()` message should be sufficient. Same for the methods below.